### PR TITLE
APIv4 PUT /commands/{command_id}

### PR DESCRIPTION
**Pull request by saturninoabril**
#### Summary
This is endpoint for `API4 PUT /commands/{command_id}`.

#### Ticket Link
n/a

#### Checklist
- [x] Added or updated unit tests (required for all new features)
- [x] Added API documentation (required for all new APIs) [mattermost-api-reference #205][205]
- [x] All new/modified APIs include changes to the drivers

[205]: https://github.com/mattermost/mattermost-api-reference/pull/205

### CODE REVIEW

File: model/client4.go [hunk 1]
```diff
@@ -2124,6 +2128,16 @@ func (c *Client4) CreateCommand(cmd *Command) (*Command, *Response) {
 	}
 }
 
+// UpdateCommand updates a command based on the provided command id string and Command struct
+func (c *Client4) UpdateCommand(commandId string, cmd *Command) (*Command, *Response) {
```
Comment (jwilander) [inline model/client4.go]: Can we just pass in the command and use `cmd.Id` for the URL building?

Comment (saturninoabril) [inline model/client4.go]: agree, please see updated PR. thanks

Comment (saturninoabril) [inline model/client4.go]: sorry, i resubmitted the PR, removing unnecessary comment used for debugging at `command.go`

Comment (jwilander) [inline model/client4.go]: No worries, thanks for the heads up 👍

**Pull request merged**